lecture/views.py

```diff
@@ -1,10 +1,21 @@
 from django.shortcuts import render
-from django.http import HttpResponse, JsonResponse
+from django.http import JsonResponse
+from django.http import HttpResponse
+from django.http import HttpResponseRedirect
 from le_dash.es import LectureWatchQuery
 
+from .forms import MpidForm
+
 
 def index(request):
-    return render(request, 'lecture/index.html')
+    if request.method == 'POST':
+        form = MpidForm(request.POST)
+        if form.is_valid():
+            return HttpResponseRedirect(form.cleaned_data['mpid'].strip())
```

Comment:
use django.urls.reverse for cases like this:

    mpid = form.cleaned_data['mpid'].strip()
    return HttpResponseRedirect(reverse('lecture', args=(mpid, )))